src/control/actor/actor.go

```diff
@@ -66,7 +66,7 @@ type Actor struct {
 	Dv5State dv5.Dv5State
 
 	GossipState     metrics.GossipState
-	GossipMetrics   metrics.GossipMetrics
+	PeerStore   metrics.PeerStore
```

Comment:
Do you think that naming it PeerStore wouldn't be misleading? There is already a `PeerStore` provided by `github.com/libp2p/go-libp2p-peerstore`. Naming can't the hardest part of the code :(((

Reply:
How about `PeerStoreMetrics`?

Reply:
Can be more suitable. However, we can also differentiate track.ExtendedPeerstore easily from metrics.PeerStore , what do you think?